apps/liteprotocoltester/infra.env

```diff
@@ -0,0 +1,11 @@
+TEST_INTERVAL_MINUTES=10
+START_PUBLISHING_AFTER=120
```

Comment:
```suggestion
START_PUBLISHING_AFTER_SECS=120
```

Reply:
Just as above, thanks for the suggestion.